ui/lib/core/addon/components/info-table-item-array.js

```diff
@@ -62,29 +63,26 @@ export default class InfoTableItemArray extends Component {
     return displayArray;
   }
 
-  async checkWildcardInArray() {
-    if (!this.args.displayArray) {
-      return;
-    }
-    let filteredArray = await this.args.displayArray.filter((item) => isWildcardString([item]));
-    this.wildcardInDisplayArray = filteredArray.length > 0 ? true : false;
-  }
-
-  @task *fetchOptions() {
+  @action async fetchOptions() {
```

Comment:
changed this to an  `@action` because concurrency didn't seem necessary since this is just called once on `did-insert` - but I haven't worked with concurrency much so if I'm missing something, please let me know!

Reply:
makes sense. we developed this during our let's use concurrency for everything phase.